dbActions.py

```diff
@@ -0,0 +1,71 @@
+import sqlite3
+import Settings
+
+
+def create_table(con):
+    """
```

Comment:
Comment should look like:
""" Create a table in case it doesn't exist

:param con
"""